src/Components/Button/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import { useState } from "react";
+import { Modal2 } from "../Modal2/Modal2";
```

Comment:
Change name to something meaningful and self-describing, `Modal2` does not make sense.

Reply:
done